bootstrap/013_task_queue.sql

```diff
@@ -14,7 +14,7 @@ wlist as (
 select count(*) > 0 as wh_exists from table(result_scan(last_query_id(-2))) where "name" = 'SUNDECK_TASK_QUEUE_WH'
 ),
 thist as (
-    select tlist.name, state, completed_time from tlist left outer join account_usage.task_history th on tlist.id = th.task_id
+    select tlist.name, state, completed_time from tlist left outer join account_usage.task_history th on tlist.id = th.task_id where th.database_name = current_database() and schema_name='TASK_QUEUE'
```

Comment:
I think this will fail -- can't use `current_database` in an owners-rights procedure. Pass in the database name from `task_queu.run_tasks_task`

Reply:
good shouit